specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2024-10-01/redisenterprise.json

```diff
@@ -0,0 +1,2145 @@
+          "readOnly": true,
+          "description": "The version of the module, e.g. '1.0'."
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "LinkedDatabase": {
+      "title": "Linked Database",
+      "description": "Specifies details of a linked database resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "arm-id",
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {
+                "type": "Microsoft.Cache/redisEnterprise/databases"
+              }
+            ]
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "description": "Resource ID of a database resource to link with this database."
+        },
+        "state": {
+          "type": "string",
+          "description": "State of the link between the database resources.",
+          "enum": [
+            "Linked",
+            "Linking",
+            "Unlinking",
+            "LinkFailed",
+            "UnlinkFailed"
+          ],
+          "x-ms-enum": {
+            "name": "LinkState",
+            "modelAsString": true
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "AccessKeys": {
+      "title": "Access keys",
+      "description": "The secret access keys used for authenticating connections to redis",
+      "type": "object",
+      "properties": {
+        "primaryKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current primary key that clients can use to authenticate"
+        },
+        "secondaryKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current secondary key that clients can use to authenticate"
+        }
+      }
+    },
+    "RegenerateKeyParameters": {
+      "title": "Regenerate access keys request",
+      "type": "object",
+      "properties": {
+        "keyType": {
+          "type": "string",
+          "description": "Which access key to regenerate.",
+          "enum": [
+            "Primary",
+            "Secondary"
+          ],
+          "x-ms-enum": {
+            "name": "AccessKeyType",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "keyType"
+      ],
+      "description": "Specifies which access keys to reset to a new random value."
+    },
+    "ImportClusterParameters": {
+      "title": "Import RDB files into a target database",
+      "type": "object",
+      "properties": {
+        "sasUris": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "SAS URIs for the target blobs to import from",
+          "x-ms-secret": true
+        }
+      },
+      "required": [
+        "sasUris"
+      ],
+      "description": "Parameters for a Redis Enterprise import operation."
```

Comment:
>Redis Enterpris [](http://example.com/codeflow?start=39&length=15)

One of the changes from the previous version had been to use RedisEnterprise as one instead of "Redis Enterprise" but i see some of the instances having the older. Suggest to make these consistent. no functional impact but consistent description.